src/Features/Core/Portable/UseAutoProperty/AbstractUseAutoPropertyAnalyzer.cs

```diff
@@ -113,6 +113,11 @@ protected void AnalyzeProperty(SemanticModelAnalysisContext context, TPropertyDe
                 return;
             }
 
+            if (property.ExplicitInterfaceImplementations.Length != 0)
```

Comment:
Should this also test the accessibility as [here](https://github.com/dotnet/roslyn/pull/23395/files#diff-92c94c254879f0d3a29d14a4615442ceR178)?

Reply:
Won't this disable the feature for VB as well as C#?